src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts

```diff
@@ -74,10 +78,77 @@ export class GeneEvidenceProteomicsComponent {
       this.selectedUniProtId = this.uniProtIds[0];
     }
 
+    this.initSRM();
+
     this.initLFQ();
     this.initTMT();
   }
 
+  initSRM() {
```

Comment:
`initSRM` looks similar to `initLFQ` and `initTMT` -- would it be possible to extract the shared logic into a helper method?

Reply:
Yes, I'll do the refactor as all three seem to share the same logic.